modules/integration/src/test/scala/scala/cli/integration/FmtTests.scala

```diff
@@ -31,19 +32,27 @@ class FmtTests extends munit.FunSuite {
     input.replaceAll("\r\n", "\n")
 
   test("simple") {
-    simpleInputs.fromRoot { root =>
+    simpleInputs("").fromRoot { root =>
       os.proc(TestUtil.cli, "fmt", ".").call(cwd = root)
       val updatedContent = noCrLf(os.read(root / "Foo.scala"))
       expect(updatedContent == expectedSimpleInputsFormattedContent)
     }
   }
 
   test("no inputs") {
-    simpleInputs.fromRoot { root =>
+    simpleInputs("").fromRoot { root =>
       os.proc(TestUtil.cli, "fmt").call(cwd = root)
       val updatedContent = noCrLf(os.read(root / "Foo.scala"))
       expect(updatedContent == expectedSimpleInputsFormattedContent)
     }
   }
 
+  test("using non-default scalafmt version") {
```

Comment:
I'm not sure, but maybe just writing unit tests would be enough. This test is very similar to "simple", and version determination can only be a unit test.